frontend/package.json

```diff
@@ -67,8 +69,8 @@
     "murmurhash-js": "1.0.x",
     "openshift-logos-icon": "1.7.1",
     "patternfly": "^3.54.8",
-    "patternfly-react": "^2.22.0",
-    "patternfly-react-extensions": "2.13.11",
+    "patternfly-react": "^2.25.1",
+    "patternfly-react-extensions": "2.13.9",
```

Comment:
I'm guessing we don't mean to downgrade this dependency?

Reply:
Nope.  Will update both to the very latest.